### PR TITLE
[JENKINS-68904] More stylelint fixes

### PR DESCRIPTION
**Pull request by NotMyFault**
Addresses the stylelint part of [JENKINS-68904](https://issues.jenkins.io/browse/JENKINS-68904), ESLint is still up to do.
Stylelint errors are no longer shifted to warnings, new introduced issues will fail the CI build.

### Proposed changelog entries

* Entry 1: Issue, Human-readable Text
* ...

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [x] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")` if applicable.
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@jenkinsci/sig-ux 

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/6792"></a>

### CODE REVIEW

File: war/src/main/less/modules/page-header.less [hunk 1]
```diff
@@ -35,8 +35,6 @@ a.page-header__brand-link {
 .page-header__brand-image {
     height: 32px;
     width: 24px;
-    height: 2rem;
-    width: 1.5rem;
```
Comment (timja) [inline war/src/main/less/modules/page-header.less:39]: are you sure the px is the intended one and not the rem? from memory last one wins?

Comment (NotMyFault) [reply war/src/main/less/modules/page-header.less]: Good catch, that should likely be a rem for the object 👍🏻

**Review by timja — COMMENTED** [review]
Thanks for this, could you push a new issue to show the build failing please? and then revert.

**Review by basil — APPROVED** [review]
Nice cleanup!

**Comment by NotMyFault** [comment]
@timja The failure can be seen on https://ci.jenkins.io/job/Core/job/jenkins/job/PR-6792/6/consoleFull, I'm going ahead and revert the commit introducing it.

**Review by timja — APPROVED** [review]

**Pull request merged**